genesyscloud/webdeployments_configuration/utils/resource_genesyscloud_webdeployments_configuration_messenger.go

```diff
@@ -75,23 +74,22 @@ func buildAppKnowledge(knowledge []interface{}) *platformclientv2.Knowledge {
 }
 
 func buildMessengerApps(apps []interface{}) *platformclientv2.Messengerapps {
-	if len(apps) < 1 {
+	if len(apps) < 1 || (len(apps) == 1 && apps[0] == nil) {
 		return nil
 	}
 
+	m := platformclientv2.Messengerapps{}
```

Comment:
Apologies, I wrote `m` as the var name to keep the code example short. I would go with a more descriptive name here, like `messengerApps`